File: app/src/main/java/com/esorokin/lantector/model/data/DetectedLanguageText.java
package com.esorokin.lantector.model.data;

import java.util.Date;

import com.esorokin.lantector.ui.adapter.DisplayableItem;

import io.requery.Entity;
import io.requery.Persistable;

@Entity
public interface DetectedLanguageText extends DisplayableItem, Persistable {
	String getText();
	String getLanguage();
	Date getDate();
}

File: app/src/main/java/com/esorokin/lantector/model/data/DetectedLanguageText.kt
package com.esorokin.lantector.model.data

import com.esorokin.lantector.ui.adapter.DisplayableItem
import io.requery.Entity
import io.requery.Persistable
import java.util.*

@Entity
data class DetectedLanguageText(val text: String = "", val language: String = "", val date: Date = Date()) : DisplayableItem, Persistable